Encodings.md

```diff
@@ -179,6 +179,12 @@ This encoding is adapted from the Binary packing described in
 ["Decoding billions of integers per second through vectorization"](http://arxiv.org/pdf/1209.2137v5.pdf)
 by D. Lemire and L. Boytsov.
 
+Delta encoding is best when used on sorted data, or data with runs of repeated
```

Comment:
Is it better to move it to the `Characteristics` section below? 

IMHO, we should not provide detail guidance like this because different users may have various opinions on this.

Reply:
I agree that this document should remain descriptive.

If we want to provide guidance and suggestions as to algorithm choice, we should create a separate section for that (in this document, or another one).

Reply:
The same thing could be said about most encodings and compression algorithms, so is that actually useful?

Reply:
Fair point. I'll just remove this.